### PR TITLE
568195 unify issued licenses storage

### PR DESCRIPTION
**Pull request by eparovyshnaya**
- generate model code for Shipments domain
- include host bundle into `lic` aggregator and `lic.shipments` feature

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.shipments.model/src-gen/org/eclipse/passage/lic/shipments/model/api/Floating.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/**
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.shipments.model/src-gen/org/eclipse/passage/lic/shipments/model/api/Floating.java:1]: We need copyright header for java files

**Review by ruspl-afed — CHANGES_REQUESTED** [review]
We need copyright header for java files

File: bundles/org.eclipse.passage.lic.shipments.model/src-gen/org/eclipse/passage/lic/shipments/model/api/Floating.java (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.shipments.model/src-gen/org/eclipse/passage/lic/shipments/model/api/Floating.java:1]: oh, shit )

Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.shipments.model/src-gen/org/eclipse/passage/lic/shipments/model/api/Floating.java]: Done

**Review by ruspl-afed — APPROVED** [review]

**Pull request merged**